PetstoreAPI/Interfaces.cs

```diff
@@ -2,18 +2,30 @@
 
 namespace PetStoreAPI
 {
-    public interface IAPIRequest
-    {
-        List<PetName> APIRequester(string url);
-    }
+    /// <summary>
+    /// Handles the running of both the APU request and the query sort, Orchestrates the program to make the whole environment flatter
+    /// </summary>
     public interface IRequestOrchestration
     {
         void Run();
```

Comment:
It's always a good idea to add some comments onto the methods themselves, so somebody calling the interface implementation knows what something is designed to do.